packages/@lwc/engine-dom/src/custom-elements/create-custom-element.ts

```diff
@@ -113,17 +113,11 @@ const createUpgradableConstructor = (
     return UpgradableConstructor;
 };
 
-export const createCustomElement = (
+export function getUpgradableConstructor(
     tagName: string,
-    upgradeCallback: LifecycleCallback,
     connectedCallback?: LifecycleCallback,
-    disconnectedCallback?: LifecycleCallback,
-    formAssociatedCallback?: LifecycleCallback,
-    formDisabledCallback?: LifecycleCallback,
-    formResetCallback?: LifecycleCallback,
-    formStateRestoreCallback?: LifecycleCallback
```

Comment:
Does this mean the FACE callbacks will not work during hydration?

Can we add a test to ensure current behavior?

Reply:
I don't think they worked before this. I can add support for them in a different PR.